golang/scripts/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
I know this was just copied from Kurt Client, but let's actually source it from https://github.com/kurtosis-tech/checklists-and-templates/blob/master/generic-bash-script.sh and modify that appropriately. We should be using that for any new Bash scripts

Reply:
Done! and I did the same for the check-markdown-links file to.